website/source/docs/guides/connect-gateways.md

```diff
@@ -0,0 +1,348 @@
+the next one.
+
+### Enable Connect in the secondary datacenter
+
+Once Connect is enabled in the primary datacenter, follow the same process to
+enable Connect in the secondary datacenter. Add the following configuration to
+the configuration for your servers, and restart them one at a time, making sure
+to maintain quorum.
+
+```
+connect {
+  enabled = true
+}
+```
+
+The `primary_datacenter` setting that was required in order to enable ACL
+replication between datacenters also specifies which datacenter will act as the
+[root CA for Connect](LINK TO MULTI-DC CONNECT DOCS) and write intentions.
+Intentions are automatically replicated to the secondary datacenter.
+
+## Deploy Gateways
+
+Connect mesh gateways proxy requests from services in one datacenter to services
+in another, so you will need to deploy your gateways on nodes that can reach
+each other over the network. You’ll also need to [generate a
+token](https://learn.hashicorp.com/consul/security-networking/production-acls#apply-individual-tokens-to-the-services)
+for each gateway that gives it read access to the entire catalog. You’ll apply
+those tokens when you start the gateways. As we mentioned in the prerequisites,
+you will need to make sure that both Envoy and Consul are installed on the
+gateway nodes. You won’t want to run any services on these nodes other than
+Consul and Envoy because they need to be more accessible than the other nodes in
+your cluster, and will have access to the WAN.
+
+### Deploy the Gateway for your primary datacenter
+
+Register and start the gateway in your primary datacenter with the following
+command.
+
+```
+consul connect envoy -mesh-gateway -register \
+                     -address "<your private address>" \
+                     -wan-address "<your externally accessible address>"\
+                     -token=<token for the primary dc gateway>
+```
+
+Next, create a [centralized configuration](LINK) file for all the sidecar
+proxies in your primary datacenter called `proxy-defaults.hcl`. This file will
+instruct the sidecar proxies to send all their inter-datacenter through the
+gateway. It should contain the following:
+
+```
+Kind = "proxy-defaults"
+Name =  "global"
+MeshGateway = "local"
+```
+
+Write the centralized configuration you just created with the following command.
+
+```
+consul config write proxy-defaults.hcl
+```
+
+### Repeat the Process to Deploy the Gateway for your Secondary Datacenter
+
+Repeat the above process for the secondary datacenter. Register and start the
+gateway in your secondary datacenter with the following command.
+
+```
+consul connect envoy -mesh-gateway -register \
+                     -address "<your private address>" \
+                     -wan-address "<your externally accessible address>"\
+                     -token=<token for the secondary dc gateway>
+```
+
+Next, create a centralized configuration file for all the sidecar proxies in
```

Comment:
Setting the proxy-defaults config entry only needs to be done once (not per-datacenter).